constraint/pkg/client/drivers/interface.go

```diff
@@ -29,6 +29,8 @@ type Driver interface {
 	RemoveTemplate(ctx context.Context, ct *templates.ConstraintTemplate) error
 	PutData(ctx context.Context, path string, data interface{}) error
 	DeleteData(ctx context.Context, path string) (bool, error)
+	AddCachedData(ctx context.Context, target string, relPath string, data interface{}) error
```

Comment:
@willbeason 

Are PutData/DeleteData going to be removed once we no longer store constraints directly in OPA?

Reply:
Yes. See https://github.com/open-policy-agent/frameworks/issues/174